models/repo.go

```diff
@@ -258,6 +258,11 @@ func (repo *Repository) APIFormat(mode AccessMode) *api.Repository {
 	return repo.innerAPIFormat(mode, false)
 }
 
+// GetCommitsCountCacheKey returns cache key used for commits count caching.
+func (repo *Repository) GetCommitsCountCacheKey(contextName string) string {
```

Comment:
This naming scheme is ambiguous. Although branch names cannot conflict with tag names, commit names may conflict with either branch or tag names (i.e. if a branch/tag's name matches a commit's SHA).

Reply:
We can't display it in repo page anyway but ok I'll add it back :)